### PR TITLE
update goquorum docs links on readme

### LINKED ISSUE #31: Links on the quorum github need updating
The [readme.md](https://github.com/ConsenSys/quorum/blob/master/README.md) on the quorum github needs updating due to the reorganisation of the documents.
All the links on the readme still point to docs.goquorum.com, which now redirects to docs.goquorum.consensys.net but not to the correct pages under the docs, so the user sees a page with "Sorry but we can't find the page you asked for".

Note that I've raised this as an issue here since it relates to the docs, but I expect this needs to be fixed on the quorum github.

### PR DESCRIPTION
**Pull request by alexandratran**
Update GoQuorum docs links on readme and add k8s logo.

fixes ConsenSys/doc.goquorum#31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,45 +5,45 @@
 [![Download](https://api.bintray.com/packages/quorumengineering/quorum/geth/images/download.svg)](https://bintray.com/quorumengineering/quorum/geth/_latestVersion)
 [![Docker Pulls](https://img.shields.io/docker/pulls/quorumengineering/quorum)](https://hub.docker.com/r/quorumengineering/quorum)
 
-Quorum is an Ethereum-based distributed ledger protocol with transaction/contract privacy and new consensus mechanisms.
+GoQuorum (formerly Quorum) is an Ethereum-based distributed ledger protocol with transaction/contract privacy and new consensus mechanisms.
```
Comment (NicolasMassart) [inline README.md]:
```suggestion
ConsenSys GoQuorum (formerly JPM Quorum) is an Ethereum-based distributed ledger protocol with transaction/contract privacy and new consensus mechanisms.
```

**Review by NicolasMassart — DISMISSED** [review]
Just a suggestion to clarify the renaming but otherwise it looks fine to me.

**Review by ricardolyn — APPROVED** [review]
LGTM

**Comment by alexandratran** [comment]
@ConsenSys/protocol-go-quorum I'm not a project collaborator, so can someone with write access merge the PR?

**Comment by ricardolyn** [comment]
@alexandratran I will do it myself. I first need change the github actions configuration to skip the linter if the change is only docs and then I will merge 👍

**Review by NicolasMassart — APPROVED** [review]

**Comment by ricardolyn** [comment]
I removed the code linters to be mandatory. there was an inconsistency between the github actions configuration and the Github merge to master configuration. For now, will merge as is but in the future we will need to review this better as PR can only be merged if the build/tests pass.

**Pull request merged**